Printer.java

```diff
@@ -0,0 +1,31 @@
+import java.util.HashMap;
+
+public class Printer {
+    
+  public void printIntroduction() {
+    System.out.println("Welcome to the great Animal Competition!");
+    System.out.println("Here you will choose your animal, customize it, and pit it in battle against another!");
+    System.out.println("First, would you like to fight a computer or a friend? (Type c for computer and f for friend");
+  }
+
+  public void describe(String described) {
```

Comment:
Why doesn't this function take an `IDescribable` as an argument and print out how to describe itself.  You shouldn't not have knowledge of specific animals within this function... this type of dependency is frowned upon.